### DISCUSSION TITLE
new version release with recent fixes?

### DISCUSSION THREAD

**Discussion by astratagem** [curl/trurl#243] [Q&A] [answered]
Hello,

Would it be possible to release a new version including changes since 0.8? Primarily I am interested in #240

The test failures mentioned in #239 are causing builds in Nixpkgs to fail (see <https://github.com/NixOS/nixpkgs/issues/264377>). We can prepare a patch to pin 3819d80e5950f3ad7a3bf097b1897cb9d392c2d8 but a new version containing this fix would be preferable.

Thank you!

**Comment by bagder** [comment, marked as answer]
That's the plan. I just want to give @jacobmealey time to sort out #235 first, then it's time to ship a 0.9.

**Reply by astratagem** [reply]
Good to know, thank you.
